src/controller/storagemgr/storagedriver/storagehandler_test.go

```diff
@@ -1,19 +1,19 @@
-/*******************************************************************************
- * Copyright 2020 Samsung Electronics All Rights Reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- *
- *******************************************************************************/
+//
+// Copyright (c) 2020 Samsung Electronics Co., Ltd All Rights Reserved.
```

Comment:
@sun-sharma One last checkup point. According to Apache License 4(b), we need to clarify prominent notices that we modified from the original file. What do you think? I mean, we should again adopt some subset of your 1st PR inc. below the line 2. (Thank you for your caveat, @tdrozdovsky) What do you think? (We need a full consent for this, since it is still not yet described in our other official guideline)

Like -
`// Copyright (c) 2020 Samsung Electronics Co., Ltd All Rights Reserved.`
`// Additional contribution on top of original storagedriver.go for Home Edge Adaptation for rest based device SDK`
`//`

Reply:
@tdrozdovsky Your point from Apache2.0 4(b) is about the re-distribution of the original code with or without modification for productization and I have not seen any `modification or something` with the copyright statement in any open source projects with Apache2.0 license. Could you give us more examples?

Meanwhile, we will check this out AGAIN with our legal council, and get back to you soon.